osquery/tables/networking/darwin/wifi.mm

```diff
@@ -99,10 +97,12 @@ void parseNetworks(const CFDictionaryRef& network, QueryData& results) {
     auto key = CFStringCreateWithCString(kCFAllocatorDefault, kv.second.c_str(),
                                          kCFStringEncodingUTF8);
     CFTypeRef value = nullptr;
-    if (key != nullptr && CFDictionaryGetValueIfPresent(network, key, &value)) {
-      r[kv.first] = extractNetworkProperties(value);
+    if (key != nullptr) {
```

Comment:
Thanks for catching this, I should've caught this one. Out of curiosity did `cppcheck` or `clang-analyze` catch this? I thought I had ran it through before the PR.